### ISSUE TITLE
Agregação política partido + regional Brasil não funciona p/ senadores e deputados

### ISSUE THREAD

**Issue by lins-morandi** [Cepesp-Fgv/cepesp-r#72]
A consulta a deputados e senadores, com agregação política = partido e agregação regional = Brasil devolve mais de uma linha para cada partido. A unidade do banco p/ esta consulta no momento é UE*Partido e não exclusivamente partido (a soma total do partido a nível nacional).
(Tudo isso na versão test e na que está aberta ao público já).

**Comment by Cepesp-Fgv** [comment]
Já tá na test.cepesp.io. Verifica se está tudo certo

**Comment by lins-morandi** [comment]
Sim, está funcionando na test p/ 2018 normalmente. Para os outros anos não consegui verificar por conta do seguinte:
![Screenshot from 2019-08-12 14-39-51](https://user-images.githubusercontent.com/47109522/62885394-26863700-bd0f-11e9-882f-e8c807f23dea.png)

Posso dar uma olhada novamente mais tarde.
